usecases/modules/vectorizer.go

```diff
@@ -352,19 +352,19 @@ func (p *Provider) addVectorToObject(object *models.Object,
 	}
 	if cfg.TargetVector() == "" {
 		object.Vector = vector
-		// TODO: add support for multi vectors
-		// object.Vector = vector
 		return
 	}
-	if object.Vectors == nil {
-		object.Vectors = models.Vectors{}
+	if multiVector != nil {
+		if object.MultiVectors == nil {
+			object.MultiVectors = models.MultiVectors{}
+		}
+		object.MultiVectors[cfg.TargetVector()] = multiVector
+	} else {
+		if object.Vectors == nil {
+			object.Vectors = models.Vectors{}
+		}
+		object.Vectors[cfg.TargetVector()] = vector
```

Comment:
**suggestion:**  is the nil checks needed, given that `Vectors` and `MultiVectors`  are maps ?

i am asking, so that we could reduce readability to 

something like that if checks not needed
```
object.Vectors[cfg.TargetVector()] = vector
if multiVector != nil {
	object.MultiVectors[cfg.TargetVector()] = multiVector
}
```

or this if needed 
```
// init empty
if object.Vectors == nil {
	object.Vectors = models.Vectors{}
}
if object.MultiVectors == nil {
	object.MultiVectors = models.MultiVectors{}
}

object.Vectors[cfg.TargetVector()] = vector
if multiVector != nil {
	object.MultiVectors[cfg.TargetVector()] = multiVector
}
```

Reply:
so the idea here is too init the `vectors` or `multiVectors` map only when it's needed and leave it `nil` that's why it's implemented this way and it's never inited empty